config/jest/jest.config.js

```diff
@@ -2,3 +2,13 @@ import Adapter from 'enzyme-adapter-react-16';
 import { configure } from 'enzyme';
 
 configure({ adapter: new Adapter() });
+
+module.exports = {
+    'testURL': 'http://localhost/',
+    'collectCoverageFrom': [
+        '!config/**/*.{js,jsx}',
+        'src/**/*.{js,jsx}',
+        '!src/**/*.Component.js',
+        '!src/_playground/**'
+    ]
```

Comment:
It's strange -- this `collectCoverageFrom` block doesn't seem to have any effect on the output of `npm run test:coverage`.  I am seeing the same result with this block, without this block or even with parts of this block modified.  I am also getting the following error when running `npm run test:coverage`:
```
> fundamental-react@0.3.5-rc.0 test:coverage /Users/I846506/dev/fundamental-react
> jest --coverage

 PASS  src/SearchInput/SearchInput.test.js
 PASS  src/Modal/Modal.test.js
 PASS  src/MultiInput/MultiInput.test.js
 PASS  src/Time/Time.test.js
 PASS  src/SideNavigation/SideNavigation.test.js
 PASS  src/InputGroup/InputGroup.test.js
 PASS  src/Shellbar/Shellbar.test.js
 PASS  src/TimePicker/TimePicker.test.js
 PASS  src/Tabs/Tabs.test.js
 PASS  src/LocalizationEditor/LocalizationEditor.test.js
 PASS  src/ComboboxInput/ComboboxInput.test.js
 PASS  src/Pagination/Pagination.test.js
 PASS  src/ActionBar/ActionBar.test.js
 PASS  src/Tree/Tree.test.js
 PASS  src/Popover/Popover.test.js
 PASS  src/Calendar/Calendar.test.js
 PASS  src/Forms/Forms.test.js
 PASS  src/DatePicker/DatePicker.test.js
 PASS  src/Breadcrumb/Breadcrumb.test.js
 PASS  src/Menu/Menu.test.js
 PASS  src/Tile/Tile.test.js
 PASS  src/Badge/Badge.test.js
 PASS  src/Alert/Alert.test.js
 PASS  src/Token/Token.test.js
 PASS  src/Table/Table.test.js
 PASS  src/Identifier/Identifier.test.js
 PASS  src/Dropdown/Dropdown.test.js
 PASS  src/Icon/Icon.test.js
 PASS  src/ListGroup/ListGroup.test.js
 PASS  src/Toggle/Toggle.test.js
 PASS  src/Panel/Panel.test.js
 PASS  src/Button/Button.test.js
 PASS  src/InlineHelp/InlineHelp.test.js
 PASS  src/Image/Image.test.js
 PASS  src/utils/CustomPropTypes/CustomPropTypes.test.js
---------------------------|----------|----------|----------|----------|-------------------|
File                       |  % Stmts | % Branch |  % Funcs |  % Lines | Uncovered Line #s |
---------------------------|----------|----------|----------|----------|-------------------|
All files                  |    98.36 |    93.37 |    96.79 |    98.57 |                   |
 config/jest               |      100 |      100 |      100 |      100 |                   |
  jest.config.js           |      100 |      100 |      100 |      100 |                   |
 src                       |      100 |      100 |      100 |      100 |                   |
  index.js                 |      100 |      100 |      100 |      100 |                   |
 src/ActionBar             |      100 |      100 |      100 |      100 |                   |
  ActionBar.js             |      100 |      100 |      100 |      100 |                   |
 src/Alert                 |      100 |      100 |      100 |      100 |                   |
  Alert.js                 |      100 |      100 |      100 |      100 |                   |
 src/Badge                 |      100 |      100 |      100 |      100 |                   |
  Badge.js                 |      100 |      100 |      100 |      100 |                   |
 src/Breadcrumb            |      100 |      100 |      100 |      100 |                   |
  Breadcrumb.js            |      100 |      100 |      100 |      100 |                   |
 src/Button                |      100 |      100 |      100 |      100 |                   |
  Button.js                |      100 |      100 |      100 |      100 |                   |
 src/Calendar              |    98.86 |     93.6 |      100 |    98.86 |                   |
  Calendar.js              |    98.86 |     93.6 |      100 |    98.86 |             35,47 |
 src/ComboboxInput         |      100 |      100 |      100 |      100 |                   |
  ComboboxInput.js         |      100 |      100 |      100 |      100 |                   |
 src/DatePicker            |    98.96 |    96.67 |      100 |    98.96 |                   |
  DatePicker.js            |    98.96 |    96.67 |      100 |    98.96 |                85 |
 src/Dropdown              |      100 |      100 |      100 |      100 |                   |
  Dropdown.js              |      100 |      100 |      100 |      100 |                   |
 src/Forms                 |      100 |      100 |      100 |      100 |                   |
  Forms.js                 |      100 |      100 |      100 |      100 |                   |
 src/Icon                  |      100 |      100 |      100 |      100 |                   |
  Icon.js                  |      100 |      100 |      100 |      100 |                   |
 src/Identifier            |      100 |      100 |      100 |      100 |                   |
  Identifier.js            |      100 |      100 |      100 |      100 |                   |
 src/Image                 |      100 |      100 |      100 |      100 |                   |
  Image.js                 |      100 |      100 |      100 |      100 |                   |
 src/InlineHelp            |      100 |      100 |      100 |      100 |                   |
  InlineHelp.js            |      100 |      100 |      100 |      100 |                   |
 src/InputGroup            |      100 |      100 |      100 |      100 |                   |
  InputGroup.js            |      100 |      100 |      100 |      100 |                   |
 src/ListGroup             |      100 |      100 |      100 |      100 |                   |
  ListGroup.js             |      100 |      100 |      100 |      100 |                   |
 src/LocalizationEditor    |      100 |      100 |      100 |      100 |                   |
  LocalizationEditor.js    |      100 |      100 |      100 |      100 |                   |
 src/Menu                  |      100 |      100 |      100 |      100 |                   |
  Menu.js                  |      100 |      100 |      100 |      100 |                   |
 src/Modal                 |      100 |      100 |      100 |      100 |                   |
  Modal.js                 |      100 |      100 |      100 |      100 |                   |
 src/MultiInput            |      100 |      100 |       95 |      100 |                   |
  MultiInput.js            |      100 |      100 |       95 |      100 |                   |
 src/Pagination            |      100 |      100 |      100 |      100 |                   |
  Pagination.js            |      100 |      100 |      100 |      100 |                   |
 src/Panel                 |      100 |      100 |      100 |      100 |                   |
  Panel.js                 |      100 |      100 |      100 |      100 |                   |
 src/Popover               |      100 |      100 |      100 |      100 |                   |
  Popover.js               |      100 |      100 |      100 |      100 |                   |
 src/SearchInput           |      100 |      100 |    95.45 |      100 |                   |
  SearchInput.js           |      100 |      100 |    95.45 |      100 |                   |
 src/Shellbar              |      100 |      100 |      100 |      100 |                   |
  Shellbar.js              |      100 |      100 |      100 |      100 |                   |
 src/SideNavigation        |    81.67 |    76.19 |    84.21 |    81.67 |                   |
  SideNavigation.js        |    81.67 |    76.19 |    84.21 |    81.67 |... 28,229,230,242 |
 src/Table                 |      100 |      100 |      100 |      100 |                   |
  Table.js                 |      100 |      100 |      100 |      100 |                   |
 src/Tabs                  |    90.91 |       75 |    90.91 |    90.91 |                   |
  Tabs.js                  |    90.91 |       75 |    90.91 |    90.91 |          40,41,46 |
 src/Tile                  |      100 |      100 |      100 |      100 |                   |
  Tile.js                  |      100 |      100 |      100 |      100 |                   |
 src/Time                  |      100 |    92.16 |    94.44 |      100 |                   |
  Time.js                  |      100 |    92.16 |    94.44 |      100 |... 49,154,174,185 |
 src/TimePicker            |      100 |    90.91 |      100 |      100 |                   |
  TimePicker.js            |      100 |    90.91 |      100 |      100 |... 92,302,319,323 |
 src/Toggle                |      100 |      100 |      100 |      100 |                   |
  Toggle.js                |      100 |      100 |      100 |      100 |                   |
 src/Token                 |      100 |      100 |      100 |      100 |                   |
  Token.js                 |      100 |      100 |      100 |      100 |                   |
 src/Tree                  |      100 |      100 |      100 |      100 |                   |
  Tree.js                  |      100 |      100 |      100 |      100 |                   |
 src/utils                 |      100 |      100 |      100 |      100 |                   |
  constants.js             |      100 |      100 |      100 |      100 |                   |
 src/utils/CustomPropTypes |    87.18 |    79.31 |     62.5 |    94.44 |                   |
  CustomPropTypes.js       |    87.18 |    79.31 |     62.5 |    94.44 |             26,61 |
---------------------------|----------|----------|----------|----------|-------------------|

Snapshot Summary
 › 30 snapshot files obsolete from 30 test suites. To remove them all, run `npm run test:coverage -- -u`.

Test Suites: 35 passed, 35 total
Tests:       16 skipped, 316 passed, 332 total
Snapshots:   30 files obsolete, 161 passed, 161 total
Time:        11.527s
Ran all test suites.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! fundamental-react@0.3.5-rc.0 test:coverage: `jest --coverage`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the fundamental-react@0.3.5-rc.0 test:coverage script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```
Anyone else seeing this?

Reply:
@greg-a-smith I'm getting that error too, I think it's just bad messaging but if you run `npm run test:coverage -- -u` to delete the obsolete snapshots the error goes away.

Reply:
@jbadan I figured removing the obsolete snapshots would help, but then I would have also thought they would have been removed as part of this PR.

Reply:
This behavior may be due to an earlier instance of this branch. I am no longer able to repo that warning when I do the following in a fresh location:

```
git clone git@github.com:SAP/fundamental-react.git fr
cd fr
npm i
npm run test

(Select "a" to run all tests. no warning about obsolete tests seen. Select "q")

 git checkout fix/clean-jest-config
rm -rf node_modules/
npm i
npm run test

(Select "a" to run all tests. no warning about obsolete tests seen. Select "q")
```

Reply:
@bcullman I'm not getting errors anymore, but I still am not getting any different result with `npm run test:coverage` with the collectCoverageFrom block existing or commented out. Is it necessary?